tests/integration/test.attachments.js

```diff
@@ -1749,7 +1749,7 @@ adapters.forEach(function (adapter) {
                 cache: true,
                 binary: true
               }, function (err, res) {
-                if (err && err.status === 405) {
+                if (err && err.status === 500) {
```

Comment:
^ what's the deal here? I know firefox had fixed the 405 bug in nightlies, but for older versions of firefox, won't this cause us to use base64 when actually blobs are okay?